README.md

```diff
@@ -205,7 +205,7 @@ The following configuration example assumes that you use the `azureblob` storage
 | processingMode            | "full"        | "full"                               | The processing mode of the LeanIX Integration API processor configuration. Defaults to partial.                                                                                                                                        |
 | lxWorkspace               | ""            | 00000000-0000-0000-0000-000000000000 | The UUID of the LeanIX workspace the data is sent to. Make sure Integration Hub data source is also setup in the same workspace                                                                                                        |
 | verbose                   | false         | true                                 | Enables verbose logging on the stdout interface of the container.                                                                                                                                                                      |
-| storageBackend            | file          | azureblob                            | The default value for the storage backend is `file`, if not provided.                                                                                                                                                                  |
+| storageBackend            | none          | azureblob                            | The default value for the storage backend is `file`, if not provided.                                                                                                                                                                  |
```

Comment:
@aravindmetku  The default value should now be 'none'  instead of 'file' right? Please update the description